--- 1.0.0.rc16/make.tmpl.in~	2009-09-16 10:55:23.000000000 +0200
+++ 1.0.0.rc16/make.tmpl.in	2015-04-22 01:02:06.498949684 +0200
@@ -136,7 +136,7 @@ $(TARGETS): $(OBJECTS)

 %.so: $(OBJECTS)
 	$(CC) -shared -Wl,-soname,$(notdir $@).$(DMRAID_LIB_VERSION) \
-	-Wl,--version-script,.export.sym $(OBJECTS) -o $@ -ldevmapper-event
+	-Wl,--version-script,.export.sym $(OBJECTS) -o $@ -ldevmapper-event -ldl

 $(LIB_STATIC): $(OBJECTS)
 	$(RM) $@
--- 1.0.0.rc16/lib/Makefile.in~	2015-04-22 01:03:23.012837544 +0200
+++ 1.0.0.rc16/lib/Makefile.in	2015-04-22 01:06:10.581617286 +0200
@@ -72,7 +72,7 @@ all: $(TARGETS)
 	@echo $(TARGETS) $(INSTALL_TARGETS)

 $(LIB_EVENTS_SHARED):
-	$(CC) events/$(LIB_EVENTS_NAME).c $(INCLUDES) $(CFLAGS) $(LDFLAGS) -shared -o $@ -ldevmapper -ldevmapper-event -lpthread $(LIB_SHARED)
+	$(CC) events/$(LIB_EVENTS_NAME).c $(INCLUDES) $(CFLAGS) $(LDFLAGS) -shared -o $@ -ldevmapper -ldevmapper-event -lpthread $(LIB_SHARED) -ldl -llvm2app

 .PHONY: install_dmraid_libs remove_dmraid_libs

