--- 1.0.0.rc16/tools/Makefile.in.sanelink~	2009-09-16 10:55:23.000000000 +0200
+++ 1.0.0.rc16/tools/Makefile.in	2012-10-29 03:37:21.002540850 +0100
@@ -72,11 +72,11 @@ all: $(TARGETS)
 
 dmraid:	$(OBJECTS) $(top_srcdir)/lib/libdmraid.a
 	$(CC) -o $@ $(OBJECTS) $(LDFLAGS) -L$(top_srcdir)/lib \
-	      -L$(DESTDIR)$(libdir) $(DMRAIDLIBS) $(DMEVENTTOOLLIBS) $(DMRAIDLIBS) $(LIBS) 
+	      $(DMRAIDLIBS) $(DMEVENTTOOLLIBS) $(DMRAIDLIBS) $(LIBS) 
 
 dmevent_tool: $(top_srcdir)/lib/libdmraid.a
 	$(CC) -o $@ $@.c $(INCLUDES) $(LDFLAGS) -L$(top_srcdir)/lib \
-	      -L$(DESTDIR)$(libdir) $(DMRAIDLIBS) $(DMEVENTTOOLLIBS) $(DMRAIDLIBS) $(LIBS) 
+	      $(DMRAIDLIBS) $(DMEVENTTOOLLIBS) $(DMRAIDLIBS) $(LIBS) 
 
 install_dmraid_tools: $(TARGETS)
 	@echo "Installing $(TARGETS) in $(sbindir)"; \
