--- 1.0.0.rc16/lib/device/scan.c.log~	2012-10-29 03:38:44.548694111 +0100
+++ 1.0.0.rc16/lib/device/scan.c	2012-10-29 03:40:05.014804857 +0100
@@ -69,7 +69,7 @@ mk_sysfs_path(struct lib_context *lc, ch
 	static char *ret = NULL, *sysfs_mp;
 
 	if (!(sysfs_mp = find_sysfs_mp(lc)))
-		LOG_ERR(lc, NULL, "finding sysfs mount point");
+		{ log_notice(lc, "finding sysfs mount point"); return NULL; }
 
 	if ((ret = dbg_malloc(strlen(sysfs_mp) + strlen(path) + 1)))
 		sprintf(ret, "%s%s", sysfs_mp, path);
@@ -315,7 +315,7 @@ discover_devices(struct lib_context *lc,
 	} else {
 		sysfs = 0;
 		path = (char *) _PATH_DEV;
-		log_print(lc, "carrying on with %s", path);
+		log_notice(lc, "carrying on with %s", path);
 	}
 
 	if (!(d = opendir(path))) {
